recipe/meta.yaml

```diff
@@ -24,97 +22,99 @@ source:
+  skip: true  # [win32 or win]
   script: |
     mkdir build
     cd build
-
-    cmake ..                                 {{ lc }}
-      -G"{{vs}}CMAKE_GENERATOR{{ve}}"        {{ lc }}
-      -DCMAKE_PREFIX_PATH="{{ prefix }}"     {{ lc }}
-      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"  {{ lc }}
-      -DCMAKE_BUILD_TYPE=Release             {{ lc }}
-      -DBUILD_EXAMPLES=OFF                   {{ lc }}
-      -DBUILD_META_EXAMPLES=OFF              {{ lc }}
-      -DBUNDLE_JSON=OFF                      {{ lc }}
-      -DBUNDLE_NLOPT=OFF                     {{ lc }}
-      -DUSE_SVMLIGHT=OFF                     {{ lc }}
-      -DENABLE_TESTING=OFF                   {{ lc }}
-      -DENABLE_COVERAGE=OFF                  {{ lc }}
-      -DUSE_SWIG_DIRECTORS=ON
-    if errorlevel 1 exit 1  # [win]
-
+    cmake ..                                                  {{ lc }}
+      -G"{{vs}}CMAKE_GENERATOR{{ve}}"                         {{ lc }}
+      -DCMAKE_PREFIX_PATH="{{ prefix }}"                      {{ lc }}
+      -DCMAKE_INSTALL_PREFIX="{{ prefix }}"                   {{ lc }}
+      -DCMAKE_BUILD_TYPE=Release                              {{ lc }}
+      -DBUILD_EXAMPLES=OFF                                    {{ lc }}
+      -DBUILD_META_EXAMPLES=OFF                               {{ lc }}
+      -DBUNDLE_JSON=OFF                                       {{ lc }}
+      -DBUNDLE_NLOPT=OFF                                      {{ lc }}
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}
+      -DENABLE_TESTING=OFF                                    {{ lc }}
+      -DENABLE_COVERAGE=OFF                                   {{ lc }}
+      -DUSE_SWIG_DIRECTORS=ON                                 {{ lc }}
+      -DLICENSE_GPL_SHOGUN=OFF                                {{ lc }}  # [bla_vendor == 'Intel']
+      -DUSE_SVMLIGHT=OFF                                      {{ lc }}  # [bla_vendor == 'Intel']
+      -DBLA_VENDOR={{ bla_vendor }}                           {{ lc }}
+      -DLAPACK_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}      {{ lc }}
+      -DBLAS_LIBRARIES={{ prefix }}/lib/{{ blas_lib }}
     cmake --build . --target shogun --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
     cmake --build . --target install --config Release -- {{ parbuild }}
-    if errorlevel 1 exit 1  # [win]
-
 requirements:
   build:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - python
-    - cmake
+    - cmake =3.15
   host:
-    - arpack       # [not win]
+    - arpack
     - bzip2
     - ctags
-    - curl
+    - libcurl
+    - libgomp       # [linux]
     - eigen
     - glpk
     - hdf5
-    - json-c       # [not win]
+    - json-c >=0.11,<=0.12
     - libxml2
     - lzo
-    - openblas     # [not win]
-    - libprotobuf  # [not win]  # conda-forge doesn't have shared lib yet
-    - rxcpp 4.0.0
+    - mkl-devel       # [bla_vendor == "Intel"]
+    - libblas {{ libblas }}    # [bla_vendor == "Generic"]
```

Comment:
which library would contain the `cblas.h` header, as currently the Generic build fails with not being able to find in the include paths `cblas.h`:
https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=127264&view=logs&j=4f922444-fdfe-5dcf-b824-02f86439ef14&t=937c195f-508d-5135-dc9f-d4b5730df0f7&l=1825

Reply:
@isuruf ?